### DISCUSSION TITLE
Welcome to Face-X Library Discussions!

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/Face-X#323] [General]
A unified library for FaceX to run all the FaceX algorithms using only one line of code.
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akshitagupta15june** [comment]
@AdityaNikhil discussions about face-x library

**Comment by akshitagupta15june** [comment]
https://github.com/akshitagupta15june/Face-X/issues/312

**Comment by shantamsultania** [comment]
hey, @AdityaNikhil I was thinking of making some wrapper classes for the flask application.
It's a bit different than what your lib was doing mine was like for example code.

def face_detection(frame):
    # does the face detection and return frame 

@app.route("/webhook")
def webhook():
    # takes input as frame 
    result = face_detection(frame)
    return result
    
   basically, I wanted to make a template to connect a OpenCV model to a Flask Application

**Reply by AdityaNikhil** [reply]
@shantamsultania Simply you mean, the result(output) will be rendered in a web app. Like in a browser, correct?

**Reply by shantamsultania** [reply]
yeah basically my idea was the same as yours at the start but as you have done it already so it will be useless to do it again so I came up with this as a change.
What's your take on this?

**Reply by AdityaNikhil** [reply]
I'd love to collaborate. I want you to first make a separate API for this. So that I'll get an idea of how your implementation looks like and then we can merge it with my API creating a unified API. We'll keep this discussion open here and share things.

**Reply by shantamsultania** [reply]
cool man will do the same
